### PR TITLE
Rewording dataframe methods attributes

### PR DESCRIPTION
**Pull request by nickynicolson**
Rewording for https://github.com/datacarpentry/python-ecology-lesson/issues/154

### CODE REVIEW

File: _episodes/01-starting-with-data.md [hunk 1]
```diff
@@ -208,22 +208,29 @@ We'll talk a bit more about what the different formats mean in a different lesso
 
 ### Useful Ways to View DataFrame objects in Python
 
-There are multiple methods that can be used to summarize and access the data
-stored in DataFrames. Let's try out a few. Note that we call the method by using
-the object name `surveys_df.method`. So `surveys_df.columns` provides an index
-of all of the column names in our DataFrame.
+There are many ways to summarize and access the data stored in DataFrames, 
+using attributes and methods provided by the DataFrame object. Attributes
```
Comment (willingc) [inline _episodes/01-starting-with-data.md]: Perhaps:

To access an attribute, use the DataFrame object name followed by the attribute name `df_object.attribute`. Using DataFrame `surveys_df` and attribute `columns`, an index of all the column names in the DataFrame can be accessed with `surveys_df.columns`.

File: _episodes/01-starting-with-data.md [hunk 2]
```diff
@@ -208,22 +208,29 @@ We'll talk a bit more about what the different formats mean in a different lesso
...
-the object name `surveys_df.method`. So `surveys_df.columns` provides an index
-of all of the column names in our DataFrame.
+There are many ways to summarize and access the data stored in DataFrames, 
+using attributes and methods provided by the DataFrame object. Attributes 
+are called using the DataFrame object name `surveys_df.attribute`, so the 
+attribute `surveys_df.columns` provides an index of all of the column names 
+in our DataFrame.
 
-## Challenges
+Methods are called using the syntax `surveys_df.method()` and we can supply extra
```
Comment (willingc) [inline _episodes/01-starting-with-data.md]: Perhaps:

Methods are called in a similar fashion using the syntax `df_object.method()`. As an example, `survey_df.head()` gets the first few rows in a DataFrame `survey_df` using the `head()` method. With a method, we can supply extra information in the parens to control behaviour.

**Comment by nickynicolson** [comment]
Thanks @willingc - added your suggestions.

**Comment by willingc** [comment]
Looks good @nickynicolson. :sunflower: @tracykteal or @wrightaprilm please merge.

**Comment by wrightaprilm** [comment]
This looks really nice, @nickynicolson. Going to go ahead and merge.

**Pull request merged**